roles/step_acme_cert/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 step_cli_executable: step-cli
+step_cli_steppath: /root/.step
```

Comment:
Similarly to the documentation feedback, I feel this should default to `$HOME/.step`.  The problem is that to actively set a reasonable default, we'd have to return to detecting the right value of `$HOME` in this file. 

Could use `ansible_env.HOME`, but that only works if facts have been gathered.  Of course, using `omit` filter won't work either, since that's a hack for excluding module arguments only.

Alternatively, we could leave the value **unset** in the defaults, and then use slightly convoluted logic in the `environment: ` block to set the value of `STEPPATH` to the empty string, but only if `step-cli` is smart enough to look for the empty string and use their built-in default.

The last option would be to use the `shell` module in place of the `command` module and explicitly set the command line to something like `STEPPATH="path" step-cli ...`, but only if the value of `step_cli_steppath` is defined.  That's gross, but might work. 

Now I'm out of ideas - curious if you have any others, or think this issue isn't worth worrying about.